plugin-server/tests/sql.test.ts

```diff
@@ -11,6 +11,8 @@ import {
 import { commonOrganizationId } from './helpers/plugins'
 import { resetTestDatabase } from './helpers/sql'
 
+jest.setTimeout(20_000)
```

Comment:
Q: Why is this needed?

Reply:
For some reason `createHub()` takes longer for me locally than in CI.